### PR TITLE
Use assimp-dev dep for building

### PR DESCRIPTION
**Pull request by cottsay**
Start moving to assimp-dev for building:

https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L106

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <build_depend>shape_msgs</build_depend>
   <build_depend>shape_tools</build_depend>
   <build_depend>octomap</build_depend>
-  <build_depend>assimp</build_depend>
+  <build_depend>assimp-dev</build_depend>
```
Comment (isucan) [inline package.xml:17]: Does this always exist? (From Ubuntu 12.04 onwards?)

Comment (cottsay) [inline package.xml:17]: Most of them existed, the rest I added in https://github.com/ros/rosdistro/commit/a506ef945febc0cd886bbc29ab976751d1cb1f3b

I don't know who created an assimp-dev rosdep entry, but it should be either used or removed.

So far Fedora is the only OS in which the dep differs.

Comment (isucan) [inline package.xml:17]: Thanks, just making sure it was in rosdistro.

**Comment by cottsay** [comment]
When Travis passes, this along with https://github.com/ros-planning/geometric_shapes/pull/14#issuecomment-33846879 would be pretty good grounds for a version bump.

This package would no longer block builds on F19 and F20 :)

Thanks!

**Comment by isucan** [comment]
Released: https://github.com/ros/rosdistro/pull/2996

**Comment by cottsay** [comment]
Thanks!

**Pull request merged**